### ISSUE TITLE
Failed to load resource

### ISSUE THREAD

**Issue by oecruz** [projectsend/projectsend#1047]
Since v1335 this si happening. The graphs never shows up

<img width="749" alt="image" src="https://user-images.githubusercontent.com/81788038/146207076-0dda4cb6-58d3-48d8-be15-69c0b48e3730.png">

Not that important, but curious about it.

**Comment by ehawman-rosenberg** [comment]
For me this functionality works on 1335 (0542a5bccbb812b168d0fcce45b37cc034d2756f).

On b9b2794abe441cfcac452882f0ff225854144a16 the statistics break 
![image](https://user-images.githubusercontent.com/81652082/147717463-04ec44e6-73c8-423a-92fb-b287ddd2bf0e.png), on b680d550d6ca975f27d384c36e5ff6b08e06fa32 the news feed breaks 
![image](https://user-images.githubusercontent.com/81652082/147717488-4e78d1d2-1c78-484c-bef8-2bfd4dc49407.png), and on 9499151e8e2504b6846be6925cd5643019294cb2 they're both broken 
![image](https://user-images.githubusercontent.com/81652082/147717513-d030dc6f-d40f-463b-b43e-8fb821227266.png)

**Comment by ehawman-rosenberg** [comment]
![image](https://user-images.githubusercontent.com/81652082/147717608-5344e363-00c3-4e04-8a99-1a3558bff7b1.png)

**Comment by ignacionelson** [comment]
Can you please check the network tab on the browser's console? The ajax calls for the widgets can be found there. I'm interested in the response tab after selecting them. Thanks!

**Comment by ehawman-rosenberg** [comment]
![image](https://user-images.githubusercontent.com/81652082/147717841-be1ad64c-cd2d-4439-b495-a53c404a9c36.png)

Not sure what I'm supposed to be looking for exactly

**Comment by ehawman-rosenberg** [comment]
[app.js response.txt](https://github.com/projectsend/projectsend/files/7791316/app.js.response.txt)

I think this is what you're asking for.

**Comment by oecruz** [comment]
It is fixed.

It gave this kind of issue in the Console in every widget and also the favicon:

_Access to XMLHttpRequest at 'https://www.site.pro/includes/widgets/ajax/statistics.php?days=15&_=1641665911321' from origin 'https://site.pro' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource._


I read a post, and it was clear that the URI was the mistake. We changed that in the General Options: System URI to be the same, and it worked again.

Seems that when we did the test the wrong URI was left in one of the clean installs. Theres just the need to adjust the Favicon, but its fine now.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
